test/deploy/k8s/deploy.sh

```diff
@@ -6,7 +6,7 @@ SCRIPT_DIR=$(dirname $0)
 
 kubectl create namespace cortx
 kubectl create cm solution-config --from-file="$SCRIPT_DIR"/cluster.yaml \
-    --from-file="$SCRIPT_DIR"/config.yaml --from-literal "node-id=123" --namespace cortx
+    --from-file="$SCRIPT_DIR"/config.yaml --from-literal "machine-id=3876243" --namespace cortx
```

Comment:
We should separate the cm for machine id, i.e. 

kubectl create cm solution-config --from-file="$SCRIPT_DIR"/cluster.yaml \
    --from-file="$SCRIPT_DIR"/config.yaml --namespace cortx

kubectl create cm node-1 --from-literal "machine-id=3876243" --namespace cortx

And use it to mount inside /etc/cortx/solution/node/ then create a link 
ln -sf /etc/cortx/solution/node/machine-id /etc/machine-id

Reply:
reverted it to node-id itself as of now. will handle machine-id implementation in separate PR